### PR TITLE
chore: update Card docs for top level recommendation

### PR DESCRIPTION
**Pull request by henrytao-me**
Part of https://github.com/Shopify/app-extension-libs/issues/980

### CODE REVIEW

**Review by alanthai — APPROVED** [review]
lgtm

File: docs/Components/Card.md [hunk 1]
```diff
@@ -10,6 +10,47 @@ Cards are used to group similar concepts and tasks together to make Shopify easi
 | ----------------------------------------------------------- | ------------------------------ |
 | Cards should be at the top level of the component hierarchy | Use too many secondary actions |
 
+If you decide to use Card in your app, it's recommended to use Card at top level without wrapping it inside any components, including Stack. If you really need something above, between, or below Card, you can wrap them in a Stack with `spacing:none` like below:
```
Comment (hannachen) [inline docs/Components/Card.md:13]:
```suggestion
If you decide to use Card in your app, it's recommended to use Card at the top level without wrapping it inside any components, including Stack. If you really need something above, between, or below Card, you can wrap them in a Stack with `spacing: none` like below:
```

File: docs/Components/Card.md [hunk 2]
```diff
@@ -10,6 +10,47 @@ Cards are used to group similar concepts and tasks together to make Shopify easi
...
+
+```ts
```
Comment (hannachen) [inline docs/Components/Card.md:15]:
```suggestion
```tsx
```

**Review by hannachen — APPROVED** [review]
Looks good, thanks for updating this! Made a few very minor suggestions.

**Pull request merged**